app.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
 const helmet = require("helmet");
 const cors = require("cors");
-const routes = require('./routes');
+const authRouter = require("./routes/auths");
 
 const app = express();
 
 app.use(helmet());
 app.use(cors());
 app.use(express.json());
-routes(app);
+// routes(app);
 
-module.exports = app;
\ No newline at end of file
+app.use("/api/auth", authRouter);
```

Comment:
@fosterferret any reason why the initial route setup didn't work for you?

Reply:
can you revert back to how the routing was done, please? We need to isolate network-related codes from the express app itself